nikohomecontrol/__init__.py

```diff
@@ -34,6 +34,13 @@ def list_locations(self):
     def execute_actions(self, id, value):
         return self._command('{"cmd":"executeactions", "id": "'+id+'", "value": "'+value+'"}')
 
+    def item_state(self, id):
+        data = self.list_actions()['data']
+        for index, item in enumerate(data):
+            if 'id' in item and str(item['id']) == str(id):
+                return item
+        return False
```

Comment:
Can you raise an error if the id is not found. So this function returns the same type.